### PR TITLE
#177 Highlights component, priority service, and current jumbotron refactor

### PR DESCRIPTION
**Pull request by morgankbender**

### CODE REVIEW

File: src/redux/getters/getTripTimes.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { getTripEvents } from './getEvents';
-import { convertAllDates } from './convertDate';
+//import { convertAllDates } from './convertDate';
```
Comment (morgankbender) [inline src/redux/getters/getTripTimes.js]: @gzinck will wait for you to merge latest PR and then resolve merge conflicts before I do anything with this branch

File: src/components/auth/login/Login.js [hunk 2]
```diff
@@ -68,7 +68,11 @@ export default class Login extends Component {
       return <Redirect to="/dashboard" />;
     } else {
       let errorDiv = '';
-      if (this.state.attemptedSubmit && !this.userInfoRetrieved()) {
+      if (
+        this.state.attemptedSubmit &&
+        !this.props.isAuthenticated &&
+        !this.props.isFetching
+      ) {
```
Comment (gzinck) [inline src/components/auth/login/Login.js]: Excellent, this fixes everything 😊

File: src/redux/getters/getTripTimes.js (hunk 1 above)
Comment (gzinck) [reply src/redux/getters/getTripTimes.js]: Sounds good!

File: src/components/auth/login/Login.js (hunk 2 above)
Comment (morgankbender) [reply src/components/auth/login/Login.js]: Yupp!! No more error as you wait for it to fetch data lol

File: src/redux/getters/getEvents.js [hunk 3]
```diff
@@ -22,18 +22,15 @@ function shouldShow(events, eventType, filter) {
...
 function isValid(state) {
-  if (
+  return !(
     state == null ||
     state.plans == null ||
     state.plans.plans == null ||
     state.accoms == null ||
     state.accoms.accoms == null ||
     state.trans == null ||
     state.trans.trans == null
```
Comment (gzinck) [inline src/redux/getters/getEvents.js]: Nice.

Comment (morgankbender) [reply src/redux/getters/getEvents.js]: Gotta love IntelliJ lol

**Pull request merged**